### PR TITLE
OCRVS-64: Add form to record an immunisation notification

### PR DESCRIPTION
**Pull request by tmvumbi2**

### CODE REVIEW

File: app/scripts/controllers/add-events.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ module.exports = function (Api, $scope, $routeParams, $location) {
   Api.Patients.get({ id: $routeParams.patientId }, success, error)
 
   $scope.events = [
-    { code: 'sample-event', display: 'Sample Event', formName: 'FormBuilderaddEventSampleEvent' }
+    { code: 'immunisation-notification', display: 'Immunisation Notification', formName: 'FormBuilderAddImmunisationNotification' }
```
Comment (kaweesi) [inline app/scripts/controllers/add-events.js]: FormBuilderAddImmunisationNotification is supposed to be initialise as a const above this file to point to your app/scripts/directives/add-events/add-event/forms/immunisation-notification.json file

**Comment by kaweesi** [comment]
thanks @tmvumbi2 just added some comment and i think we need to add some tests in  test/services/events.js, try mimicking what @BMartinos has at: https://github.com/jembi/opencrvs-immunisation-app/pull/5/files
please take a look at this PR @BMartinos too!

File: app/scripts/directives/add-events/add-event/index.js [hunk 2]
```diff
@@ -32,6 +32,10 @@ module.exports = function (loadResource, $q, state, FHIR, FormBuilderService) {
                   setProcedureEventType(encounterTemplate, scope.event.code, 'Sample Event')
                   resourceTemplateDict = { main: encounterTemplate }
                   break
+                case 'immunisation-notification':
+                  setProcedureEventType(encounterTemplate, scope.event.code, 'Immunisation Notification')
```
Comment (kaweesi) [inline app/scripts/directives/add-events/add-event/index.js]: how about replacing 'Immunisation Notification' with scope.event.display

File: app/scripts/directives/add-events/add-event/forms/immunisation-notification.json [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+          "checks": []
+        },
+        "options": [
+          {"key": "Chuk", "value": "Chuk"},
+          {"key": "Kacyiru Police Hospital", "value": "Kacyiru Police Hospital"},
+          {"key": "Muhima Hospital", "value": "Muhima Hospital"},
+          {"key": "Kibagabaga Hospital", "value": "Kibagabaga Hospital"},
+          {"key": "Kanombe Military Hospital", "value": "Kanombe Military Hospital"},
+          {"key": "King Faisal Hospital", "value": "King Faisal Hospital"},
+          {"key": "Ndera Hospital", "value": "Ndera Hospital"},
```
Comment (kaweesi) [inline app/scripts/directives/add-events/add-event/forms/immunisation-notification.json]: just thinking these keys ought to be diverse from the values/names

File: app/scripts/controllers/add-events.js [hunk 4]
```diff
@@ -35,7 +35,8 @@ module.exports = function (Api, $scope, $routeParams, $location) {
   Api.Patients.get({ id: $routeParams.patientId }, success, error)
 
   $scope.events = [
-    { code: 'birth-notification', display: 'Birth Notification', formName: 'FormBuilderAddEventBirthNotification' }
+    { code: 'birth-notification', display: 'Birth Notification', formName: 'FormBuilderAddEventBirthNotification' },\
```
Comment (kaweesi) [inline app/scripts/controllers/add-events.js]: whAT ABOUT the ending backward slash on this line!!!

Comment (napiergit) [inline app/scripts/controllers/add-events.js]: fixed here: https://github.com/jembi/opencrvs-immunisation-app/pull/14/files

**Pull request merged**